### PR TITLE
Add macOCR

### PR DESCRIPTION
**Pull request by jaklan**
## Description

Add OCR with `macOCR`

## Type of change

- [x] New script command
- [ ] Bug fix
- [ ] Improvement of an existing script
- [ ] Documentation update
- [ ] Toolkit change
- [ ] Other (Specify)

## Dependencies / Requirements

[macOCR](https://github.com/schappim/macOCR)

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/productivity/macocr/macocr-run-ocr.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Dependencies:
+# macOCR: https://github.com/schappim/macOCR
+
+# Recommended installation:
+# brew install schappim/ocr/ocr
```
Comment (unnamedd) [inline commands/productivity/macocr/macocr-run-ocr.sh:7]: Nice you added the instructions here, would you mind to follow the way defined in the [contributing.md](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)? 
```
# Dependency: This script requires `jq` cli installed: https://stedolan.github.io/jq/
# Install via homebrew: `brew install jq`
```

File: commands/productivity/macocr/macocr-run-ocr.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+# Optional parameters:
+# @raycast.packageName macOCR
+# @raycast.icon 📸
+# @raycast.argument1 { "type": "text", "placeholder": "Language (default: en-US)", "optional": true }
+
+# Documentation:
+# @raycast.author Jakub Lanski
+# @raycast.authorURL https://github.com/jaklan
+
+result=$(ocr -l ${1:-"en-US"})
```
Comment (unnamedd) [inline commands/productivity/macocr/macocr-run-ocr.sh:23]: Also, make sure to check the existence of the `ocr` installed in the system:

```
if ! command -v download &> /dev/null; then
      echo "download command is required (https://github.com/kevva/download-cli).";
      exit 1;
fi
```

File: commands/productivity/macocr/macocr-run-ocr.sh [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+echo $result | tee >(pbcopy)
```
Comment (unnamedd) [inline commands/productivity/macocr/macocr-run-ocr.sh:24]: I guess `tee` isn't installed by default in the system, right? If yes, it's better to add up in the requirements and also in the check before use.

**Review by unnamedd — CHANGES_REQUESTED** [review]
Hey @jaklan, thanks for you contribution (again <3)!

I've made some small comments, please let me know when you finish to address them!

File: commands/productivity/macocr/macocr-run-ocr.sh (hunk 3 above)
Comment (jaklan) [reply commands/productivity/macocr/macocr-run-ocr.sh:24]: @unnamedd hm, afaik - it is installed.
```shell
❯ which tee
/usr/bin/tee
```

Comment (dehesa) [reply commands/productivity/macocr/macocr-run-ocr.sh:26]: It is also in my machine

**Comment by dehesa** [comment]
Thanks for the addition @jaklan

**Pull request merged**